src/test/isolation2/sql/resgroup/resgroup_cancel_terminate_concurrency.sql

```diff
@@ -115,4 +115,28 @@ SELECT * FROM rg_concurrency_view;
 DROP ROLE role_concurrency_test;
 DROP RESOURCE GROUP rg_concurrency_test;
 
+-- test5: terminate a query waiting for a slot, that opens a local transaction on exit callback
```

Comment:
It is an incorrect comment - we open a regular global transaction, not a local one. I'll fix it later.